src/components/shared/BookMark.tsx

```diff
@@ -0,0 +1,28 @@
+import * as S from "./BookMark.styled";
+
+interface Props {
+  ownerInfo: {
+    name?: string;
+    owner?: { [key: string]: string | undefined };
+  };
+}
+
+const BookMark = ({ ownerInfo }: Props) => {
+  if (!ownerInfo) {
+    return;
+  }
+  const { name, owner } = ownerInfo;
+
+  return (
+    <div>
+      <S.OwnerInfoWrap>
+        <img src={owner && owner.profileImageSource} alt="소유자 프로필" />
```

Comment:
owner 가 없는 경우에는 src 에 undefined 라 들어가게 되어서, 좀 더 안전하게 없는 경우에는 빈 문자열이라도 넣어주시는 걸 추천드려요!
```suggestion
        <img src={owner?.profileImageSource ?? ''} alt="소유자 프로필" />
```